mantidproject/mantid
# Add normalisation by bin width for 2d plots in workbench

**Description of work.**
This PR as normalisation by bin width in the same way it is implemented for 1d plots. 
In Mantidplot, spectrum view rebins the data before plotting so it is always normalised. 

Note: This has not been done implemented in sliceviewer, this will be done in a separate PR (#27359 ) 

**To test:**
Load a workspace in workbench (e.g. MAR11060)
Right click on the workspace and click `plot`-> `colorfill`
Right click on the plot and toggle the normalisation and check it works as expected (you can compare with mantidplot to check if it is correct) 
Try doing 2d plots from the script and check they can be normalised correctly (example scripts can be found [here](https://docs.mantidproject.org/nightly/api/python/mantid/plots/index.html))

Fixes #27468 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
